Source/SourceKittenFramework/Module.swift

```diff
@@ -202,3 +192,67 @@ private extension Collection where Element == XcodeBuildSetting {
         return lazy.compactMap(getterClosure).first
     }
 }
+
+// MARK: Yams.Node helpers for SwiftPM
+
+// The yaml structure changed in Xcode 15.3 / SwiftPM 5.10.  This extension decodes both formats.
+private extension Node {
```

Comment:
You can define Swift version `fiveDotTen` in [SwiftVersion](https://github.com/jpsim/SourceKitten/blob/main/Source/SourceKittenFramework/SwiftVersion.swift) and pick the correct selection path depending on this version here instead of trying both.

Reply:
I don't think so: `SwiftVersion` is about the version of Swift providing SourceKit whereas this code is about some version of Swift whose SwiftPM created a `debug.yaml`.